packages/training/book/src/operations/local-training.md

```
@@ -0,0 +1,274 @@
+- vLLM process: 20-35% VRAM
+- Total: 70-85% VRAM
+
+### Weights & Biases (Optional)
+
+```bash
+# Enable W&B
+export WANDB_API_KEY=your_key
+python scripts/run_training.py --profile 12gb
+```
+
+View at: https://wandb.ai/your-team/babylon-training
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)